cockroachcloud/cloud-sso.md

```diff
@@ -73,8 +73,8 @@ Yes, as long as Enterprise Authentication is enabled for your {{ site.data.produ
 
 ### Which authentication flows are supported with Enterprise Authentication? Is it possible to enable the identity provider initiated flow?
 
-Enterprise Authentication currently supports only a Service Provider (CockroachDB Cloud) initiated flow. The identity provider initiated flow can not be enabled yet.
+{{ site.data.products.enterprise }} Authentication supports only a flow initiated by {{ site.data.products.db }}. Initiation from the IdP is not supported.
 
-### What default role is assigned to users when autoprovisioning is enabled in a CockroachDB Cloud org?
+### What default role is assigned to users when auto-provisioning is enabled in a {{ site.data.products.db }} organization?
 
-`Developer role` is assigned by default to autoprovisioned users.
+The `Developer` role is assigned by default to autoprovisioned users.
```

Comment:
Please either use "auto-provisioning" everywhere or add "autoprovisioning" to the dictionary file and use that everywhere.

Reply:
done.